src/IOManagement.cpp

```diff
@@ -0,0 +1,73 @@
+#include "IOManagement.h"
+#include "const.h"
+#include <Arduino.h>
+#include "STM32TimerInterrupt_Generic.h"
```

Comment:
Let's move all of these includes (except for `IOManagement.h`) into `IOManagement.h`, if they aren't already there.